### DISCUSSION TITLE
how can I apply my changes to css and js?

### DISCUSSION THREAD

**Discussion by solo10010** [thuliteio/doks#481] [Q&A] [answered]
Hi @h-enk I'm facing another problem!

- how can I apply my changes to css and js?

I made edits to /node_modules/@huas/*
maybe it's wrong, but I didn't find any more places...

when I start the server "npm run server" then everything is applied. but when I fill in git push all styles remain untouched.
I tried to start doing "npm run build" but no changes.

I will sum up the results

1. the main question is how to apply changes in the files /node_modules/@huas/* to the template
2. maybe it's worth removing.gitignore?

Thanks for your attention!

**Comment by h-enk** [comment, marked as answer]
Child theme files override parent theme files.

So, for the files you'd like to change, copy the folder-structure + files from  `./node_modules/@hyas/doks/assets/` to `./assets/`. In the latter directory, update the files.

**Reply by solo10010** [reply]
gh-pages solution to the problem with workflows

**I will identify the problem**

I didn't have any workflows running, I didn't understand why. the gh-pages branch was not created!

**The cause of the problem is thoughtless Ctrl-C Ctrl-V**
git creates a repository with the master main branch, but we push in main,
and master is registered in the `workflows` files for quick installation
, so if `push` in main, the workflows will not start!
the problem is in the command below! it's better to just `git push`
```bash
git push -u origin main
```

**To Do**

* create your own repository for the layout, for example, `mysite`
* clone your empty `git clone repository https://github/youname/mysite`
* `git clone https://github.com/h-enk/doks .git my-doks-site`
* copy all contents of 'my-docks-site' to mysite except files.git and .github
* create a folder in our repo `.github/workflows` and the `deploy-github.yml`  file in it
* copy to a file
```bash
# Deploy your Hyas site to GitHub Pages

name: GitHub Pages

on:
push:
branches:
- master

jobs:
deploy:
runs-on: ubuntu-20.04
steps:
- uses: actions/checkout@v2
- uses: actions/setup-node@v2
with:
node-version: '16'
cache: 'npm'

- name: Install dependencies
run: npm install

# - name: Check for linting errors
# run: npm test

- name: Build production website
run: npm run build

- name: Deploy to GitHub Pages
uses: peaceiris/actions-gh-pages@v3
with:
github_token: ${{ secrets.GITHUB_TOKEN }}
publish_dir: ./public
```
* bays everything to your git just `git push`
